legacy/src/api/assets/tokens.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+grand_parent: API Reference
+parent: Assets
+title: Tokens
+permalink: /api/tokens
+---
+
+# Tokens
+{:.no_toc}
+
+Tokens are assets which can be spent only once. They are usually tied to a
+small set of merchants and have an expiry date.
+
+Tokens belong to a collection. A collection defines the redemption rules and
+branding for all tokens under the collection.
+
+Token value may be set in multiple currencies and is the same for all tokens in the same collection.
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Create Token
+
+{% reqspec %}
+  POST '/api/tokens'
+  auth 'api-key'
+  body ({
+    accountId: 'Te2uDM7xhDLWGVJU3nzwnh',
+    ledgerId: 'centrapay.nzd.main',
+  })
+{% endreqspec %}
+
+{% h4 Fields %}
+
+|     Field      |  Type   |                                      Description                                       |
+| :------------- | :------ | :------------------------------------------------------------------------------------- |
+| accountId      | String  | The account that will own the new token.                                               |
+| collectionId   | String  | The token collection that will govern the branding and redemption rules for the token. |
+| idempotencyKey | String  | Client-supplied identifier that prevents double creation.                              |
+| liveness       | Boolean | 'main' if the token is a live token, 'test' if the token is a test token.              |
+
+{% h4 Example response payload %}
+
+{% json %}
+{
+  "id": "pe32dd90-b46c-11ea-92828sa",
+  "accountId": "WRhAxxWpTKb5U7pXyxQjjP",
+	"category": "token",
+	"collectionId": "Jaim1Cu1Q55uooxSens6yk",
+  "status": "active",
+  "createdAt": "2021-01-17T18:00:23.000Z",
+  "validFrom": "2021-01-17T18:00:23.000Z",
+  "expiresAt": "2022-01-18T18:00:23.000Z",
+	"liveness": "test",
+	"createdBy": "crn:1234abc:api-key:MyAssetIssuerKey",
+	"description": "My Test Token",
+	"value": [{
+		"currency": "NZD",
+		"amount": 400
+	}],
+	"type": "centrapay.token.test"
+}
+{% endjson %}
+
+{% h4 Error Responses %}
+
+| Status |              Code               |                             Description                              |
+| :----- | :------------------------------ | :------------------------------------------------------------------- |
+| 403    | TOKEN_ALREADY_CREATED           | Token with supplied parameters already exists.                       |
+| 403    | TOKEN_COLLECTION_DOES_NOT_EXIST | No collection can be found with the supplied collectionId.           |
```

Comment:
This should be a generic 403 https://docs.centrapay.com/api/http-status-codes#resource-missing-or-permission-denied

Reply:
I've removed this from the table :)